operator/gitops/argocd/pipeline-service/openshift-pipelines/namespace.yaml

```diff
@@ -0,0 +1,9 @@
+---
+apiVersion: v1
```

Comment:
Can you add a comment to record why that needs to happen?
I'm worried we might ask ourselves in a few months and not remember why.

Reply:
Added a comment pointing to the Jira issue as I found we did in other cases.